app/views/buyers/applications/new.html.erb

```diff
@@ -20,10 +20,7 @@
                           :default_plan => @plans.respond_to?(:default) ? @plans.default : @plans.first %>
 
     <% if current_account.settings.service_plans.allowed? %>
-      <%= form.input :service_plan_id, as: :select, collection: [], label: ServicePlan.model_name.human,
-        hint: link_to('Create a service plan', admin_services_path, id: 'link-help-new-application-service') %>
```

Comment:
I haven't figured out what we mean with this hint…

Reply:
me neither 😂 but it doesn't make any sense to me and I didn't see it in the page when I loaded it 🤔

Reply:
right, me neither